src/include/btree.h

```diff
@@ -12,7 +12,13 @@
 #define WT_BTREE_MAJOR_VERSION_MIN 1 /* Oldest version supported */
 #define WT_BTREE_MINOR_VERSION_MIN 1
 
+/* Increase the version number for standalone build. */
+#ifdef WT_STANDALONE_BUILD
+#define WT_BTREE_MAJOR_VERSION_MAX 2 /* Newest version supported */
+#else
 #define WT_BTREE_MAJOR_VERSION_MAX 1 /* Newest version supported */
+#endif
+
```

Comment:
My understanding was the we were updating the version numbers (which will update the files we create), but we won't check the version numbers unless we're in a standalone build?

@hbokhari, wasn't that the case?

Reply:
@kommiharibabu

Reply:
As per my understanding from the last discussion, we will update the version number and check it only when the compatibility option is not provided for WT standalone mode.

If we want the version number to be changed by default and only check in case of standalone mode and don't verify it for MongoDB release will have problems as we need to fix all the older versions of MongoDB to not to check for the version number. Due to this, we went with update the version number also only for standalone mode.

Please let me know if it doesn't align with your view?

Reply:
@kommiharibabu, I understand now, thank you.

I hate to let this go, because I find it uncomfortable that it's impossible to tell the format from the file, that seems like a bad idea.

We'd only have to backport to a single release, I think, as only the last release can seen "new" files. That said, I won't argue strongly, I take your point.

cc: @agorrod

Reply:
I had expected that MongoDB wouldn't generate the new version number, per the change here.
We can't generate newer version numbers in MongoDB databases, since it would require adding a new downgrade floor (as Hari described) to expect the newer btree ID version.